### PR TITLE
Add linux_arm-64 to supported platforms for shareccclasses jvmti test

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Resolves https://github.com/eclipse/openj9-systemtest/issues/90 
Signed-off-by: Mesbah_Alam@ca.ibm.com <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
This PR allows system tests to build on aarch64 platform. Compilation of test tested here: Test_grinder/job/Grinder_Advanced/1749/console

**Comment by Mesbah-Alam** [comment]
Hi @smlambert , could you please review and merge this PR?

File: openj9.test.sharedClasses.jvmti/src/native/Makefile [hunk 1]
```diff
@@ -66,7 +66,7 @@ P=:
 ###
 # Check platform is set to a single valid value
 ###
-VALID_PLATFORMS?=aix_ppc-32,aix_ppc-64,linux_390-31,linux_390-64,linux_ppc-32,linux_ppc-64,linux_x86-32,linux_x86-64,win_x86-32,win_x86-64,zos_390-31,zos_390-64,linux_ppcle-64,linux_arm-32,linux_aarch64-64,osx_x86-64
+VALID_PLATFORMS?=aix_ppc-32,aix_ppc-64,linux_390-31,linux_390-64,linux_ppc-32,linux_ppc-64,linux_x86-32,linux_x86-64,win_x86-32,win_x86-64,zos_390-31,zos_390-64,linux_ppcle-64,linux_arm-32,linux_aarch64-64,osx_x86-64,linux_arm-64
```
Comment (smlambert) [inline openj9.test.sharedClasses.jvmti/src/native/Makefile]: Is linux_aarch64-64 unused then?  Or do we have a situation where there are multiple names for same things in different places?  I am fine to merge this, but want to check if we can remove the other, or if both are currently used.

I see that this addresses this failure: https://ci.adoptopenjdk.net/view/Test_system/job/Test_openjdk11_j9_extended.system_aarch64_linux/48/console

Comment (Mesbah-Alam) [reply openj9.test.sharedClasses.jvmti/src/native/Makefile]: `java_platform` is returning `linux_arm-64` - which is set in top.xml by using Java System properties.  

As in https://ci.adoptopenjdk.net/view/Test_system/job/Test_openjdk11_j9_extended.system_aarch64_linux/48/consoleFull

```18:47:08       [echo] make -C /home/jenkins/workspace/Test_openjdk11_j9_extended.system_aarch64_linux/openjdk-tests/system/openj9-systemtest/openj9.test.sharedClasses.jvmti/src/native SRCDIR=/home/jenkins/workspace/Test_openjdk11_j9_extended.system_aarch64_linux/openjdk-tests/system/openj9-systemtest/openj9.test.sharedClasses.jvmti/src/native **PLATFORM=linux_arm-64** 
```

I do not see `java_platform` being set to `linux_aarch64-64` in any aarch64 builds (checked jdk 8, 11, 13 system test builds at adopt). 

It appears Java system properties was previously returning  `linux_aarch64-64` as the value for platform and now switched to  `linux_arm-64` ? I am not sure.

Comment (Mesbah-Alam) [reply openj9.test.sharedClasses.jvmti/src/native/Makefile]: A test should be to remove `linux_aarch64-64` - and run grinders in jdk 8, 11, 13 on aarch64 builds and make sure it compiles

Comment (Mesbah-Alam) [reply openj9.test.sharedClasses.jvmti/src/native/Makefile]: Also, it's noteworthy that `linux_aarch64-64` doesn't exist in the modularity test's native compile script: https://github.com/AdoptOpenJDK/openjdk-systemtest/blob/master/openjdk.test.modularity/src/tests/com.test.jlink/native/makefile 

We have `linux_arm-64` there - and it compiles fine in JDK11 and 13 on both Adopt and OpenJ9 JDKs.

**Comment by Mesbah-Alam** [comment]
Testing the updated PR (after removing `linux_aarch64-64`) on OpenJ9 JDK 11 : /Test_grinder/job/Grinder/1990/console

**Comment by Mesbah-Alam** [comment]
The test compiled fine in /Test_grinder/job/Grinder/1990/console. 

@smlambert - I think we can merge this PR at this point. it will replace `linux_aarch64-64` with `linux_arm-64`.

**Pull request merged**